src/include/common/TDigest.h

```diff
@@ -0,0 +1,675 @@
+/*
+ * Licensed to Derrick R. Burns under one or more
+ * contributor license agreements.  See the NOTICES file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Modified by Sangtian Wang (wsangtian@vmware.com) */
```

Comment:
Wondering what's the part you modified, is it possible to add the dependency of the original one instead of copy past the whole file?